app.js

```diff
@@ -1,7 +1,22 @@
 const express = require("express")
 const app = express()
-const {getApi} = require("./controllers/api.controller")
+const {getApi, getTopics} = require("./controllers/api.controller")
 
 app.get("/api", getApi)
+app.get("/api/topics", getTopics);
+
+
+app.use((req, res, next) => {
+    res.status(404).send({ msg: "not found" });
```

Comment:
At the moment, we will never reach the final app.use where the server error lives - what code do we need to add here to make sure this happens in the event that we don't want to send a 404?